.github/workflows/haskell.yml

```diff
@@ -160,7 +160,7 @@ jobs:
     - name: Archive working directory
       # pax format is needed to avoid unnecessary rebuilding, because the
       # default format uses a one-second resolution for timestamps
-      run: tar --use-compress-program zstdmt --format pax -cf /var/tmp/state.tzst . && mv /var/tmp/state.tzst .
+      run: tar --use-compress-program zstdmt --format posix --exclude-vcs -cf /var/tmp/state.tzst . && mv /var/tmp/state.tzst .
```

Comment:
Note that `pax` and `posix` are different names for the same format. I just think `posix` looks less weird.